<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QtCharts::QChartView" name="wChart" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Keithley 2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Keithley 2614B</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName">
       <property name="text">
        <string>KEITHLEY2614</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStart">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
